gui/build.rs

```diff
@@ -8,6 +9,23 @@ const LINUX_INCLUDE: &str = r#"
 "#;
 
 fn main() {
+    if std::env::var("CARGO_FEATURE_GUI_SLINT").is_ok_and(|var| var == "1") {
+        build_bin();
+    } else if std::env::var("CARGO_FEATURE_QT_FFI").is_ok_and(|var| var == "1") {
+        build_lib();
+    }
+}
+
+fn build_bin() {
+    // Compile main
+    slint_build::compile_with_config(
+        "slint/main.slint",
+        CompilerConfiguration::new().with_style(String::from("fluent-dark")),
+    )
+    .unwrap();
```

Comment:
Better to recursive enumerate all files in this directory and compile all of `.slint` files.

Reply:
We will only be compiling one file. Slint doesn't support multiple files.